packages/juggler/src/legacy.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: juggler
```

Comment:
I think this should be `@loopback/juggler`? Please fix the module name in all new files if needed.

Also, can we use a more descriptive file name than `legacy.ts`? For example `legacy-dao-bridge.ts`.